wire-compiler/src/main/java/com/squareup/wire/schema/WireRun.kt

```diff
@@ -145,13 +145,16 @@ data class WireRun(
     val schemaLoader = NewSchemaLoader(fs, sourcePath, protoPath)
     val protoFiles = schemaLoader.load()
 
-    // 2. Validate the schema and resolve references
-    val fullSchema = Linker(protoFiles).link()
+    // 2. Load descriptor proto
+    val protoFilesPlusDescriptor = protoFiles.plus(schemaLoader.loadDescriptorProto())
```

Comment:
could have put this in NewSchemaLoader#load, but felt side-effecty and broke current tests.  Open to moving it back.